uint8 ptp_lock_offset
uint8[16] ptp_status